Packs/CVE_2022_30190/Playbooks/playbook-CVE-2022-30190_-_MSDT_RCE.yml

```diff
@@ -1296,26 +1302,113 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  '42':
```

Comment:
I didn't saw the playbook yet but want to make sure - do we have an input whether should we run the XQL?

Reply:
I see we have it, great!

Reply:
fixed